action.yml

```diff
@@ -0,0 +1,18 @@
+name: pysa
+description: "Run pysa on the repository and check for vulnerabilities"
+
+inputs:
+  build-type:
+    description: Pysa build to use. Possible values are stable and nightly. Default is stable.
+    required: false
+    default: "stable"
+runs:
+  using: "composite"
+  steps:
+    - run: if [ ${{inputs.build-type}} = "nighty" ]; then pip3 install pyre-check-nightly; else pip3 install pyre-check; fi
+      shell: bash
+    - run: if [ ! -f .pyre_configuration ]; then printf ".\n" | pyre init; fi
+      shell: bash
```

Comment:
We can try to run `pyre infer` before launching pysa. With this pyre will try to infer more type information and Pysa may be able to find more issues https://pyre-check.org/docs/pysa-coverage/#pyre-infer

Reply:
Yeah, we can try that!

Reply:
Did it!